src/OAuth2/ResponseType/JwtAccessToken.php

```diff
@@ -127,5 +127,15 @@ protected function createPayload($client_id, $user_id, $scope = null)
             'token_type' => $this->config['token_type'],
             'scope'      => $scope
         );
+        
+        if (isset($this->config['jwt_extra_payload']) && is_callable($this->config['jwt_extra_payload'])) {
+            $func = $this->config['jwt_extra_payload'];
+            $extra = $func($client_id, $user_id, $scope);
+            if (is_array($extra)) {
+                $payload = array_merge($extra, $payload);
```

Comment:
I don't see why can't change those if we need. specification does not say they are required and even if this lib does require some when we override them here it is most likely we will handle this change in other places as well.
@bshaffer what do you think?